### PR TITLE
[DOC] odoo_analyse: add extra documentation info

### PR DESCRIPTION
**Pull request by Yenthe666**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,11 @@ $ pip3 install "odoo-analyse[graph]"
 
 `-s /path/to/data.json` .. Store the loaded modules in a file
 
+Or if you want to output it to `stdin`:
+
+`-s -` .. Outputs the loaded modules to stdin
```
Comment (fkantelberg) [inline README.md:31]: It's stdout twice. Could you also add the `--load - .. Load the modules from stdin` in the section above?

**Review by fkantelberg — CHANGES_REQUESTED** [review]
Just a small typo and request for completion.

File: README.md (hunk 1 above)
Comment (Yenthe666) [reply README.md:31]: Euhm, how do you mean it is `stdout` twice?
& about `--load`: hmm, I don't see to get this command honestly 🤔

Comment (fkantelberg) [reply README.md:31]: You output to stdout and not stdin as described in the PR.

`-l/--load` can load files created by `-s/--save`. This was initially done to add a cache. E.g. if you analyse an entire Odoo instance it takes quite long. But if you want to see different graphs afterwards you have to run it twice. This is the reason we implemented `-s` and `-l` in the first place. Analyse bigger repositores/Odoo installations and save it into a file. Afterwards you can create your statistic out of the file which is fast or share the analysed data.

**Comment by Yenthe666** [comment]
@fkantelberg changed

File: README.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ $ pip3 install "odoo-analyse[graph]"
 
 `-s /path/to/data.json` .. Store the loaded modules in a file
 
+Or if you want to output it to `stdout`:
+
+`-s -` .. Outputs the loaded modules to stdout
+
+### Loading files
+`-l /path/to/data.json` .. Loads the JSON data from a file which you can use for detailed analysis/graphs.
+This serves as a cache-like system to analyse repositories or Odoo installations by first saving the data into a file and then reusing it.
```
Comment (fkantelberg) [inline README.md:35]: This section is already there. Only a note about the usage of `-` to load the cache from stdin would be cool.

**Comment by Yenthe666** [comment]
I guess this is better :)

**Comment by fkantelberg** [comment]
Yes thank you. I'll have a look to describe some examples in the next days.

**Comment by Yenthe666** [comment]
Thanks 🙏

**Pull request merged**